RELEASE_NOTES.md

```diff
@@ -3,6 +3,8 @@
 ## in develop
 
 * Docker image dependencies that are DNAnexus platform files are included in applet's bundledDepends
+* Adds `-instanceTypeSelection` compiler option to allow disabling compile-type instance type selection
```

Comment:
Ok, so this would still be optional and the alternative would be to cache the instance type DB based on what we got from the rank API during compile?

Reply:
yes - the default is "static" which means to attempt compile-time determination of instance types and to store the instance type database in the applet metadata